development/docker-compose/readme.md

```diff
@@ -144,11 +159,11 @@ The following is an example configuration:
 
 ### Ingest data
 
-The following tasks ingest data:
-- Product catalog task
-- Legal entity bootstrap task
+The bootstrap-job ingests the following data:
```

Comment:
I wouldn't add any link to https://github.com/baas-devops-reference since the vast majority of developers don't have access to it in Backbase, same for the ones outside backbase